partcad/src/partcad/part_factory_feature_ai.py

```
@@ -657,13 +661,13 @@ def _render_image(self, script, candidate_id):
         def render(part):
             nonlocal exception_text
             try:
-                coro = part.get_shape()
+                coro = part.get_wrapped(self.ctx)
                 with pc_logging.Action("Instantiate", part.project_name, part.name):
                     shape = asyncio.run(coro)
                 if not shape is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)